doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -18,6 +18,29 @@ For detailed information about supported functionalities, see the official `Fast
    The implementation passes tests of `Fast Pair Validator app`_ (beta version).
    The procedure triggered in Android settings is successful (tested with Android 11).
 
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with Fast Pair, make sure that the following prerequisites are completed:
+
+* :ref:`Install the nRF Connect SDK <installation>`.
+* Setup of the required :term:`Development Kit (DK)`.
```

Comment:
```suggestion
* Setup of the supported :term:`Development Kit (DK)`.
```
tbc

Reply:
I have made this a bit clear. It is about setting up of DK (mainly steps mentioned in the GSG of a DK).